npm_and_yarn/spec/fixtures/npm_responses/types_jquery.json

```diff
@@ -0,0 +1 @@
githubUsername":"choffmeister"},{"name":"Steve Fenton","url":"https://github.com/Steve-Fenton","githubUsername":"Steve-Fenton"},{"name":"Diullei Gomes","url":"https://github.com/Diullei","githubUsername":"Diullei"},{"name":"Tass Iliopoulos","url":"https://github.com/tasoili","githubUsername":"tasoili"},{"name":"Jason Swearingen","url":"https://github.com/jasons-novaleaf","githubUsername":"jasons-novaleaf"},{"name":"Sean Hill","url":"https://github.com/seanski","githubUsername":"seanski"},{"name":"Guus Goossens","url":"https://github.com/Guuz","githubUsername":"Guuz"},{"name":"Kelly Summerlin","url":"https://github.com/ksummerlin","githubUsername":"ksummerlin"},{"name":"Basarat Ali Syed","url":"https://github.com/basarat","githubUsername":"basarat"},{"name":"Nicholas Wolverson","url":"https://github.com/nwolverson","githubUsername":"nwolverson"},{"name":"Derek Cicerone","url":"https://github.com/derekcicerone","githubUsername":"derekcicerone"},{"name":"Andrew Gaspar","url":"https://github.com/AndrewGaspar","githubUsername":"AndrewGaspar"},{"name":"Seikichi Kondo","url":"https://github.com/seikichi","githubUsername":"seikichi"},{"name":"Benjamin Jackman","url":"https://github.com/benjaminjackman","githubUsername":"benjaminjackman"},{"name":"Poul Sorensen","url":"https://github.com/s093294","githubUsername":"s093294"},{"name":"Josh Strobl","url":"https://github.com/JoshStrobl","githubUsername":"JoshStrobl"},{"name":"John Reilly","url":"https://github.com/johnnyreilly","githubUsername":"johnnyreilly"},{"name":"Dick van den Brink","url":"https://github.com/DickvdBrink","githubUsername":"DickvdBrink"},{"name":"Thomas Schulz","url":"https://github.com/King2500","githubUsername":"King2500"},{"name":"Terry Mun","url":"https://github.com/terrymun","githubUsername":"terrymun"}],"main":"","types":"index","repository":{"type":"git","url":"https://github.com/DefinitelyTyped/DefinitelyTyped.git","directory":"types/jquery"},"scripts":{},"dependencies":{"@types/sizzle":"*"},"typesPublisherContentHash":"df62dc4dca38a444be365d9e0268e088a19ae08d0a4464c56083e0aea9df2c54","typeScriptVersion":"2.3","_id":"@types/jquery@3.3.30","dist":{"integrity":"sha512-chB+QbLulamShZAFcTJtl8opZwHFBpDOP6nRLrPGkhC6N1aKWrDXg2Nc71tEg6ny6E8SQpRwbWSi9GdstH5VJA==","shasum":"af4ad612d86d954d74664b2b0ec337a251fddb5b","tarball":"https://registry.npmjs.org/@types/jquery/-/jquery-3.3.30.tgz","fileCount":10,"unpackedSize":1099089,"npm-signature":"-----BEGIN PGP SIGNATURE-----\r\nVersion: OpenPGP.js v3.0.4\r\nComment: https://openpgpjs.org\r\n\r\nwsFcBAEBCAAQBQJdESoWCRA9TVsSAnZWagAABPwP/iDJYNJv02A06PbtuX2L\nX8gyF+bfBkt7I0fTsA3EuGghiWUeVFc6ANxx8RzE6WLPnuoy7w8GNCanUhjd\n6n8FtxUpKRbZbEumFmrM7uQmBE4rz+7e/TXndXsDd9sCYo5by1hSET6xjLNf\n3QV3FzSjjgVZP7ou0REHT4jMFNowbY7bqcsW2eN6fDOs57L2X9+UwmpOczP/\nYpi1TLD8Qdkso0RAT7oFpG4FiRXhHe1kZtshwfB9Ndnrnwd5O/yLLQtkMWbV\nye250DVz1fNmrMEyKBVM3D2t3XuMJt9TmgqxKpmEPmUdNOSNvnBfxMK4egCM\nrKuM9KNLbYpI1WHYV7M8udfULUrjIKYxjiTra7cy0KnjMovAbFEe3cbbD4Ia\n7wiw3GOWtsSSJDnj3T7Kqi3Vg
```

Comment:
Should this be empty?

Reply:
What are you referring to here?  Just not clear on the context of this comment.

Reply:
Sorry for the confusion 😅 

I was referring to the fixture file only having a new line. I noticed the other fixture did as well. I didn't delve deeply into the code, so I'm not sure how the file is being parsed/processed. I think I had expected to find some data in the fixture.